apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/Program.cs

```diff
@@ -0,0 +1,513 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Google.Api.Gax;
+using Google.Cloud.Spanner.V1.Internal.Logging;
+
+namespace Google.Cloud.Spanner.Data.YCSB
+{
+    internal class Program
```

Comment:
Do we know whether YCSB is likely to be useful for other APIs? I'm just wondering whether it's worth separating the Spanner-specific parts from the non-Spanner-specific parts. Maybe not, or at least not now - just worth considering.

Reply:
I was actually thinking this myself and this would be a good thing.  I'll do the refactor to two classes but for now still keep it all in one assembly.